{"name": "Test03: Verify Sending Email to Parabank Support", "status": "broken", "statusDetails": {"message": "AttributeError: module 'selenium.webdriver.edge.webdriver' has no attribute 'Edge'", "trace": "request = <SubRequest 'init_web_driver' for <Function test_verify_login_error_parabank[nir-12344]>>\n\n    @pytest.fixture(scope='class')\n    def init_web_driver(request):\n        if get_data('Execute_Applitools').lower() == 'yes':\n>           globals()['driver'] = get_web_driver()\n\ntests_cases\\conftest.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntests_cases\\conftest.py:115: in get_web_driver\n    driver = get_edge()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    def get_edge():\n        # srv = Service(EdgeChromiumDriverManager().install())  # selenium 4\n        # edge_driver = selenium.webdriver.Edge(service=srv)  # selenium 4\n>       edge_driver = webdriver.Edge(EdgeChromiumDriverManager().install()) # selenium 3\nE       AttributeError: module 'selenium.webdriver.edge.webdriver' has no attribute 'Edge'\n\ntests_cases\\conftest.py:148: AttributeError"}, "description": "This test verifies that an email successfully sent to Parabank customer support.", "start": 1734965095736, "stop": 1734965095736, "uuid": "37bc7663-007a-4f30-a1a5-2d55e4152cf7", "historyId": "6fbcbfea8a4bc77728e7d5895fd7c78b", "testCaseId": "6fbcbfea8a4bc77728e7d5895fd7c78b", "fullName": "tests_cases.test_web.TestWeb#test_verify_sending_email_to_support", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_web_driver')"}, {"name": "parentSuite", "value": "tests_cases"}, {"name": "suite", "value": "test_web"}, {"name": "subSuite", "value": "TestWeb"}, {"name": "host", "value": "DESKTOP-DHGB96Q"}, {"name": "thread", "value": "11592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests_cases.test_web"}]}